added Taego, Paramo and Haven to map_names and dimensions

eiriksanderfjeld opened the pull request.

rlohne committed:
added Taego, Paramo and Haven to map_names and dimensions

rlohne committed:
added Taego, Paramo and Haven to map_names and dimensions

crflynn commented:
Thanks for adding these maps. I'm happy to add them but I'm not sure about adding CI here. The existing tests are limited and relatively weak and/or broken. It probably needs some work before we add CI.

eiriksanderfjeld commented:
Right, will remove it and update the PR. 

How have you run the tests previously?

eiriksanderfjeld commented:
@crflynn Something else you want me to fix?

crflynn commented:
> How have you run the tests previously?

I would just run them locally. Admittedly I didn't put as much effort into this project as I could have.

> Something else you want me to fix?

No, sorry I forgot about this. Taking a look now.

crflynn commented on chicken_dinner/constants.py:
```diff
@@ -67,6 +73,9 @@
     "Summerland_Main": [204800, 204800],
+    "Tiger_Main": [819200, 819200],
+    "Chimera_Main" : [819200, 819200],
+    "Heaven_Main" : [307200, 307200],
```
I've not played this game in a long time. Is this correct? Some googling shows this is a 1x1 map.

rlohne replied:
No, there is a typo here. I will fix.

rlohne replied:
I've fixed the typos; Chimera_Main (Paramo) should be 3x3, and Heaven (Haven) is 1x1. Paramo is a bit of a strange one though; the map locations (towns and other named locations) rotate from match to match. Names etc. stay the same, but move to different parts of the map. Kinda like School from Erangel suddenly being on the island. Not sure how well that will work with replays, but the coordinates stay the same I should think; just that using the same basemap will make for some wierd renders. 

Going to have  a look at chicken_dinner.assets as well, and then I will commit the changes.

crflynn commented:
We might also want to update the assets here. There is some code in `chicken_dinner.assets` you can run or I can do it separately.

rlohne commented:
> We might also want to update the assets here. There is some code in `chicken_dinner.assets` you can run or I can do it separately.

I'll take a look at it; there are some new weapons etc.

rlohne committed:
updated to reflect changes in patch 12.2
order is:
vehicleId.json
damageTypeCategory.json
damageCauserName.json
itemId.json

rlohne committed:
fixed typos

Merge branch 'master' of https://github.com/Team-PUBG/chicken-dinner

# Conflicts:
#	chicken_dinner/constants.py

eiriksanderfjeld commented:
@crflynn Something else we should look into?
